TYPO3.Neos/Documentation/CreatingASite/ContentCache.rst

```diff
@@ -182,6 +183,19 @@ The ``ContentCollection`` cache configuration declares a tag that will flush the
 any of it's descendants (direct or indirect child) changes. So editing a node inside the collection will flush the
 whole collection cache entry and cause it to re-render.
 
+.. note::
+  When using ``cached`` as the cache mode, your ``entryTags`` should always contain the node identifier. Otherwise, the
+  cache will not be flushed when you make changes to the node itself, which will lead to unexpected behavior in the Neos
```

Comment:
That's right, for nodes that only render static output you would not need it. But you probably wouldn't do a custom cache setting for these, would you? (Trying to understand your use case ;) )